airflow/providers/google/cloud/hooks/looker.py

```diff
@@ -0,0 +1,308 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""This module contains a Google Cloud Looker hook."""
+
+import json
+import time
+from enum import Enum
+from typing import Dict, Optional
+
+# TODO: Temporary import for Looker API response stub, remove once looker sdk 22.2 released
+import attr
+from looker_sdk.rtl import api_settings, auth_session, requests_transport, serialize
+# TODO: Temporary import for Looker API response stub, remove once looker sdk 22.2 released
+from looker_sdk.rtl.model import Model
+from looker_sdk.sdk.api40 import methods as methods40
+from packaging.version import parse as parse_version
+
+from airflow.exceptions import AirflowException
+from airflow.hooks.base import BaseHook
+from airflow.models.connection import Connection
+from airflow.version import version
+
+# TODO: uncomment once looker sdk 22.2 released
+# from looker_sdk.sdk.api40.models import MaterializePDT
+
+
+@attr.s(auto_attribs=True, init=False)
+class MaterializePDT(Model):
+    """
+    TODO: Temporary API response stub. Use Looker API class once looker sdk 22.2 released.
```

Comment:
Do we have some release date? Maybe we can wait if it's soon?

Reply:
same here - I removed these TODOs completely as looker sdk 22.2 is finally launched now.